<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuFooter</class>
 <widget class="QWidget" name="MenuFooter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Background" native="true">
     <property name="styleSheet">
      <string notr="true">#Background
{
background-color: #F7F8FA;
}</string>
     </property>
     <widget class="KelGuiPushButton" name="btnSettings">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>0</y>
        <width>132</width>
        <height>132</height>
       </rect>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="style">
       <enum>KelGuiPushButton::Settings</enum>
      </property>
     </widget>
     <widget class="KelGuiPushButton" name="btnConnection">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>0</y>
        <width>132</width>
        <height>132</height>
       </rect>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="style">
       <enum>KelGuiPushButton::Connection</enum>
      </property>
     </widget>
     <widget class="KelGuiPushButton" name="btnAccaunt">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>0</y>
        <width>132</width>
        <height>132</height>
       </rect>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="style">
       <enum>KelGuiPushButton::Account</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>188</y>
        <width>134</width>
        <height>5</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
background-color: #000000;
border-radius: 2px;
}

*::chunk
{
background-color: #05B8CC;
border-radius: 2px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KelGuiPushButton</class>
   <extends>QPushButton</extends>
   <header>kelguipushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
